src/pages/guides/index.md

```diff
@@ -1,57 +1,115 @@
+[Modules](../guides/modules/index.md) are mini-editing experiences that you can offer to your users. They are perfect for focused creative tasks without loading the full editor.
 
-### What are modules?
+Available modules:
 
-[Modules](../guides/modules/index.md) are contained mini-editing experiences that you can surface to your users. The modules available at this time are **Generate Image** and **Edit Image**.
+- Generate image: AI-powered tools to create unique visuals instantly.
+- Edit image: Tools for photo retouching, applying filters, and adjusting backgrounds.
 
-## Getting Started
+## Developer journey
+
+1. **Get an API key**: Refer to the [quickstart](./quickstart/index.md) guide.
+2. **Use documentation**: Utilize our [code sample](https://github.com/AdobeDocs/cc-everywhere/tree/main/v4-sample/sample.md) and [demo app](https://demo.expressembed.com/) code snippets.
+3. **Submit for review**: Submit your integration for [review](./review/index.md) by Adobe.
+4. **Share publicly**: Once approved, share the integration with your users!
+
+## Additional resources
+
+<DiscoverBlock slots="link, text"/>
+
+[Quickstart Guide](../guides/quickstart/)
+
+Learn how to get started with the Adobe Express Embed SDK.
 
 <DiscoverBlock slots="link, text"/>
 
-[Quickstart](./quickstart/index.md)
+[Tutorials](..//guides/tutorials/)
 
-Read our quickstart to begin integrating with the Adobe Express Embed SDK.
+Explore the collection of tutorials.
 
 <DiscoverBlock slots="link, text"/>
 
-[Community forum](https://community.adobe.com/t5/adobe-express-embed-sdk/ct-p/ct-express-embed-sdk?page=1&sort=latest_replies&lang=all&tabid=all)
+[Sample Code](https://github.com/AdobeDocs/cc-everywhere/tree/main/v4-sample)
 
-Follow to get announcements, debug issues, and submit feature requests.
+Adobe Express Embed SDK Sample using V4 SDK.
+
+<DiscoverBlock slots="link, text"/>
 
-## Developer Journey
+[Community](https://developer.adobe.com/express/community)
 
-1. Create a project in [Adobe Developer Console](https://adobe.io/console). Refer to the [quickstart](./quickstart/index.md) guide for these steps.
-2. Use this documentation, [code sample](https://github.com/AdobeDocs/cc-everywhere/tree/main/v4-sample/sample.md) and [demo app](https://demo.expressembed.com/) code snippets to help you get started with development.
-3. Submit your integration for [review](./review/index.md) by Adobe.
-4. Once approved, share the integration publicly with your users!
+If you need assistance or have questions about the Adobe Express Embed SDK, visit the community section for access to community forums.
```

Comment:
Why not a link here to the community section, too...

Reply:
This is linked, it is right after the discover block.